### PR TITLE
Label animation and determind if selected items has changed

### PR DESCRIPTION
**Pull request by althurzard**
Implement label animation just like react-native-material-textfield 
Implement onChangeSelectedItems function returns selectedItems

### CODE REVIEW

File: src/react-native-selectize.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { StyleSheet, Text, TextInput, TouchableOpacity, View, ViewPropTypes } fr
 import PropTypes from 'prop-types';
 import { normalize, schema } from 'normalizr';
 import Chip from './react-native-chip';
-
+import Label from './label'
```
Comment (BukhariMuslim) [inline src/react-native-selectize.js]: What this line refer to? I cannot find that file.

Comment (c-moss-talk) [reply src/react-native-selectize.js]: @althurzard did you forget to add the Label component `index.js` to your commit?

**Comment by althurzard** [comment]
oops, i forgot adding label folder :D.  I'll create another pull request.

File: src/react-native-selectize.js (hunk 1 above)
Comment (althurzard) [reply src/react-native-selectize.js]: oops, i forgot adding label folder :D. I'll create another pull request later.

**Comment by c-moss-talk** [comment]
@BukhariMuslim , I tested this and works beautifully for me! Can we merge? thanks @althurzard !

**Comment by raynor85** [comment]
Thank you for the pull request. I've checked your modification with the emailValidation example: the graphic looks broken and the onChipSelect stopped working. Can you please fix it? I'd like to merge it but I don't want to introduce other issues.

**Pull request closed without merging**